docs/tutorials/kafka-integration.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 18
+title: Integrating with Kafka
+description: Learn how to integrate DBOS and Kafka.
+---
+
+In this guide, you'll learn how to use DBOS transactions and workflows to process Kafka messages with exactly-once semantics.
+
+First, define your transaction or workflow. It must take in the Kafka topic, partition, and message as inputs:
+
+```javascript
```

Comment:
Is this the best for ts?

Reply:
I also tried `typescript`, but it didn't work very well.